Samples/Server/Unity-FractalStream/Assets/Scripts/WebRTCServer.cs

```diff
@@ -352,21 +417,31 @@ private void OnInputData(string data)
                         break;
 
                     case "camera-transform-stereo":
-
                         string camera = node["body"];
-                        if (camera != null && camera.Length > 0)
+                        if (!cameraNeedUpdated && camera != null && camera.Length > 0)
                         {
                             var coords = camera.Split(',');
-                            var index = 0;
-                            for (int i = 0; i < 4; i++)
+
+                            // Parse the prediction timestamp from the message body.
+                            long timestamp = long.Parse(coords[32]);

```

Comment:
this makes the server only compatible with timestamp capable clients - can we use a different message format to facilitate both 1) old clients that do not send a timestamp and 2) new clients that do?

Reply:
Good catch! I think that we'll need to add a new message type (for example **camera-transform-stereo-prediction**).